app/managers.py

```diff
@@ -0,0 +1,52 @@
+import sqlite3
+from typing import List
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self,
+               first_name: str,
+               last_name: str
+               ) -> None:
+        query = f"""
+        INSERT INTO {self.table_name} (first_name, last_name)
+        VALUES (?, ?);
+        """
+        self._connection.execute(query, (first_name, last_name))
+        self._connection.commit()
+
+    def all(self) -> List[Actor]:
+        cursor = self._connection.execute(

```

Comment:
The variable name `cursor` is not very descriptive. Consider renaming it to something more intuitive like `result_cursor` or `actor_cursor` to better convey its role in fetching data from the database. This will improve code readability and maintainability.